test/test_torch.py

```diff
@@ -12513,10 +12513,6 @@ def rand_tensor(size, dtype, device):
             else:
                 alpha = 3
 
-            # addcmul is not supported for complex dtypes on cuda yet
-            if device.startswith('cuda') and dtype.is_complex:
-                continue
-
             actual = torch.addcmul(a, b, c, value=alpha)
             expected = a + alpha * b * c
```

Comment:
I'm confused -- why isn't there a `self.assertEqual(actual, expected)` check here?

Reply:
nvm, it looks like github is having problems showing the code: it skips straight from like 12521 to 13436